pkg/resource/table/hooks.go

```diff
@@ -418,9 +419,17 @@ func customPreCompare(
 	a *resource,
 	b *resource,
 ) {
-	// TODO(hilalymh): customDeltaFunctions for AttributeDefintions
+	// TODO(hilalymh): customDeltaFunctions for AttributeDefinitions
 	// TODO(hilalymh): customDeltaFunctions for GlobalSecondaryIndexes
 
+	if aws.StringValue(a.ko.Spec.BillingMode) == string(v1alpha1.BillingMode_PAY_PER_REQUEST) {
```

Comment:
Might be good to add a little comment here:

```go
// See https://github.com/aws-controllers-k8s/community/issues/1595
```

to help developers in the future understand this.

Reply:
Hi @jaypipes  thanks for feedback,  add comment in [37d63a5](https://github.com/aws-controllers-k8s/dynamodb-controller/pull/55/commits/37d63a5d3a4f2da91e00ee13b06602f41bd53194)